CMakeLists.txt

```diff
@@ -185,12 +185,63 @@ endif()
 #add_subdirectory(iverilog)
 add_subdirectory(libs)
 add_subdirectory(libopenfpga)
-add_subdirectory(yosys)
+#add_subdirectory(yosys)
 add_subdirectory(abc)
 add_subdirectory(ace2)
 add_subdirectory(vpr)
 add_subdirectory(openfpga)
 
+# Compilation options for yosys
+include(CMakeParseArguments)
+
+##project(yosys)
+
+# Version number 
+set(YOSYS_VERSION_MAJOR 0.7)
+set(YOSYS_VERSION_MINOR 0)
+set(YOSYS_VERSION_PATCH 0)
+
+# Options to enable/disable dependencies
+option(YOSYS_ENABLE_TCL, "Enable TCL parser integrated in yosys" ON)
```

Comment:
Our local sever has a problem in setting up TCL library. I would suggest turn it off by default. As such, we can avoid modifying it locally

Reply:
Made this change.